.cfnlintrc.yaml

```diff
@@ -134,8 +134,13 @@ ignore_templates:
   - tests/translator/output/**/function_with_tracing.json # Obsolete DependsOn on resource
   - tests/translator/output/**/api_with_propagate_tags.json # TODO: Intentional error transform tests. Will be updated.
   - tests/translator/output/**/function_with_intrinsics_resource_attribute.json # CFN now supports intrinsics in DeletionPolicy
+  - tests/translator/output/**/embedded_connectors_hardcoded_props.json # Only checking if the embedded connectors work
```

Comment:
Why does this test need to be excluded?

Reply:
oops, meant to fix that test but forgot. Let me fix it